src/wp-includes/script-loader.php

```diff
@@ -1233,8 +1236,16 @@ function wp_default_scripts( $scripts ) {
 		$scripts->add( 'inline-edit-tax', "/wp-admin/js/inline-edit-tax$suffix.js", array( 'jquery', 'wp-a11y' ), false, 1 );
 		$scripts->set_translations( 'inline-edit-tax' );
 
-		$scripts->add( 'plugin-install', "/wp-admin/js/plugin-install$suffix.js", array( 'jquery', 'jquery-ui-core', 'thickbox' ), false, 1 );
+		$scripts->add( 'plugin-install', "/wp-admin/js/plugin-install$suffix.js", array(), false, 1 );
 		$scripts->set_translations( 'plugin-install' );
+		did_action( 'init' ) && $scripts->localize(
+			'plugin-install',
+			'pluginL10n',
+			array(
+				'close'            => __( 'Close' ),
+				'noiframes'        => __( 'This feature requires inline frames. You have iframes disabled or your browser does not support them.' )
```

Comment:
To fix PHP error, there just needs to be a comma at the end of this line.

Reply:
Thanks for the explanation! Now done!